tests/integration/components/doi-contributor-test.js

```diff
@@ -3,12 +3,13 @@ import { setupRenderingTest } from 'ember-qunit';
 import { setupFactoryGuy, make } from 'ember-data-factory-guy';
 import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
+import { skip } from 'ember-qunit';
 
 module('Integration | Component | doi contributor', function(hooks) {
   setupRenderingTest(hooks);
   setupFactoryGuy(hooks);
 
-  test('it renders', async function(assert) {
+  skip('it renders', async function(assert) {
```

Comment:
Is this part also for review ?

Reply:
Yes.  Please review this PR again including these 2 tests.  I have removed the 'skip' so those tests are run.  However, I had to change the tests because the assert.dom('*') is not picking up all of the text in the doi-contributor and doi-creator components.  The text is rendered, but it is just not being picked up in by the selector in the test.  I have tried a number of things to get it to work properly but none of them work.  Thanks.